diff --git a/plugins/coroae/uwsgiplugin.py b/plugins/coroae/uwsgiplugin.py
index 28ecfac..0449b37 100644
--- a/plugins/coroae/uwsgiplugin.py
+++ b/plugins/coroae/uwsgiplugin.py
@@ -14,7 +14,7 @@ if not coroapi:
 
 NAME='coroae'
 CFLAGS = os.popen('perl -MExtUtils::Embed -e ccopts').read().rstrip().split()
-CFLAGS += ['-Wno-int-to-pointer-cast', '-Wno-error=int-to-pointer-cast', '-I%s/Coro' % coroapi]
+CFLAGS += ['-Wno-format', '-Wno-int-to-pointer-cast', '-Wno-error=int-to-pointer-cast', '-I%s/Coro' % coroapi]
 LDFLAGS = []
 LIBS = []
 
